network/network.go

```diff
@@ -1052,3 +1006,61 @@ func (n *network) disconnected(p *peer) {
 		n.router.Disconnected(p.id)
 	}
 }
+
+type PeerElement struct {
```

Comment:
The peer struct already has the ID, so I'm not sure the PeerElement struct is necessary

Reply:
I believe I need to keep this struct to hold the sent bool.  So we can properly log based on stephen's comments..
I did however see that the validator id is the key, and I use peer.id now.  (thx)